src/util.cpp

```diff
@@ -1847,6 +1847,7 @@ QCString removeRedundantWhiteSpace(const QCString &s)
           *dst++=' '; // add extra space after
         }
         break;
+      case '[':
```

Comment:
This will process `A[]` as `A []`. I afraid this might have unwanted site-effects.

Reply:
I do understand, any suggestions how to tackle the problem as when we have:
```
	/** {@link #main(String[])}
	 *  {@link #foo(Object...)}
	 * @param args */
	public static void main(String[] args) {}
```

at one place in the original the `String` and `[]` are separated and in the other place they are not.

Reply:
I've committed an alternative fix: https://github.com/doxygen/doxygen/commit/3360831da443551791f4115cd4c5373b6f99a8d7